src/base/File.h

```diff
@@ -115,10 +115,10 @@ class File
     // Windows-specific HANDLE is needed because LockFileEx() does not take POSIX FDs.
 #if _SQUID_WINDOWS_ || _SQUID_MINGW_
     typedef HANDLE Handle;
-    static const Handle InvalidHandle;
+    static constexpr Handle InvalidHandle = INVALID_HANDLE_VALUE;
```

Comment:
AFAICT, theses changes must be propagated to the following InvalidHandle definition in src/base/File.cc (removing the latter, I guess):

```C++
#if _SQUID_WINDOWS_ || _SQUID_MINGW_
const HANDLE File::InvalidHandle = INVALID_HANDLE_VALUE;
#endif /* _SQUID_WINDOWS_ || _SQUID_MINGW_*/
```